### ISSUE TITLE
Add co-speaker feature broken

### ISSUE THREAD

**Issue by mariobehling** [fossasia/eventyay-talk#129]
When an organiser activates "additional speaker" in the content section the form for speakers shows two areas to enter a speaker email. 

a) The top form field works as expected. 
b) When the speaker adds an email in the second form field nothing happens after saving the form.

This is confusing. Why are there two areas?

However in contrast to eventyay-talk on Pretalx this seems to work. Identify the difference and fix the issue to work in the same way as Pretalx.

Organiser content form
![Screenshot from 2024-07-19 00-52-50](https://github.com/user-attachments/assets/7d264473-4c7e-4309-8b34-d0942c8ce7c0)

Speaker form
![Screenshot from 2024-07-19 00-49-04](https://github.com/user-attachments/assets/bef47f75-cfd9-4e61-b166-62fa8016866e)

Tested on https://wikimania-dev.eventyay.com/

**Comment by odkhang** [comment]
Hi @mariobehling ,

I have checked Pretalx and see it work the same as eventyay-talk

1. As the first page when you submit a proposal, and fill in 'Additional Speaker', invitation will be send without any issue

![image](https://github.com/user-attachments/assets/017c333a-5916-412a-a226-8561e647ba1b)

2. But in the edit proposal, it correct as you mentioned, additional Speaker not work.


 => the addtional Speaker only work once when we submit proposal

To prevent the confusing, there're 2 options:

1. Remove 'Additional Speaker' as we can use this 
![image](https://github.com/user-attachments/assets/3aa851b4-7960-46b9-ad67-d0dfd9571ae1)

2. Send invitation for 'Additional Speaker' if it added

**Comment by mariobehling** [comment]
@odkhang @lcduong Yes, I think option 2 would make sense the most for the moment: Send invitation for 'Additional Speaker' if it added

**Comment by lcduong** [comment]
sure @mariobehling, please help me to review this PR: https://github.com/fossasia/eventyay-talk/pull/130

**Comment by hpdang** [comment]
The issue was solved. The co-speaker feature works as expected
